test/integration/corechecks/docker/main_test.go

```diff
@@ -26,9 +26,6 @@ var retryDelay = flag.Int("retry-delay", 1, "time to wait between retries (defau
 var retryTimeout = flag.Int("retry-timeout", 30, "maximum time before failure (default 30 seconds)")
 var skipCleanup = flag.Bool("skip-cleanup", false, "skip cleanup of the docker containers (for debugging)")
 
-// Must be repeated in the following dockerCfgString
-const instanceTag = "instanceTag:MustBeHere"
```

Comment:
I'd rather keep the instance tag in the integration test, as the bigger the scope, the most likely we are to catch a regression. What's the reason you're removing it?

Reply:
As the tags are now added at the end by the Sender, it's actually right to not have the instance tag as this point, for example, this Gauge method should indeed not be called with the instance tags:

https://github.com/DataDog/datadog-agent/pull/2723/files#diff-3eb9762119461430618d0b121dd4887cL21

The test ensuring that the instance tags are correctly added to the send would now be this one, in the Sender:
https://github.com/DataDog/datadog-agent/pull/2723/files#diff-8cdf7cfd035fe412835107a034b1ec3eR137

Let me know what you think about it @xvello , I may have miss something important conceptually. I'm very open to change things if needed.

Reply:
Indeed, my bad!